content/docs/start/data-management.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Data Management Trail
```

Comment:
does it makes sense to take the current get started (remove metrics, remove experiments, remove even pipelines?) and make it a data management trail for now? it covers already a lot of things there ...

Reply:
I also believe that's the most sensible way to start. I'll start by some _rearrangement._